### ISSUE TITLE
A11y: speak to those who cannot read

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpoints#291]
See [WP#31368](https://core.trac.wordpress.org/ticket/31368).

**Comment by JDGrimes** [comment]
I'm hesitant to pursue this so late in the game here. I wish that we had done so while working on the new Hooks UI, but we didn't. Let's punt.

**Comment by JDGrimes** [comment]
I'm not sure what, if anything, should be done in regard to our dynamic selectors. So for the moment at least I'm just going to focus on the other stuff.

**Comment by JDGrimes** [comment]
Looking for any hints from WordPress core on how to handle the dynamic selectors, I discovered that when a user chooses a term, it [uses the `assertive` priority](https://core.trac.wordpress.org/browser/trunk/src/wp-admin/js/tags-suggest.js?rev=38880#L105), rather than the default, which is `polite`. A look at the [WC3 guidelines](https://www.w3.org/TR/wai-aria/states_and_properties#aria-live) suggests that assertive value "unless the interruption is imperative." I guess maybe the delay is partly affected by typing, which would is why being assertive makes sense in the context of the term input box. However, I think that for the most part we can afford to be polite here. Anyway, we can always change this later if users find that they are not receiving the messages in a timely manner due to the normal flow.

**Comment by JDGrimes** [comment]
When performing a search of the available widgets in the customizer, [the number of results is announced](https://core.trac.wordpress.org/browser/trunk/src/wp-admin/js/customize-widgets.js?rev=38709#L251), but I think that is different than our dynamic selectors, since they aren't really a search.

So let's call this fixed for now. We'll open new issues for improvements that we think are needed in the future.
